x-pack/plugin/security/qa/operator-privileges-tests/src/javaRestTest/java/org/elasticsearch/xpack/security/operator/Constants.java

```diff
@@ -424,6 +424,7 @@ public class Constants {
         "indices:data/read/async_search/submit",
         "indices:data/read/close_point_in_time",
         "indices:data/read/eql",
+        "indices:data/read/entsearch",
```

Comment:
I believe this can be removed as well

Reply:
Can you please help me understand why this configuration was needed at first and why it's no longer needed?

Reply:
I think we added a new privilege for Enterprise Search on the PoC. It is not needed now as we haven't yet dealt with authZ or created a new endpoint

Reply:
Thanks for the explanation. I'm happy to remove it.